NHS.Api/README.md

```diff
@@ -7,12 +7,12 @@ INSTALLATION :-
 Run requirement27.txt file.
 
 API's :-
-Search : "http://<Domain Name>:<port_number>/<bucket_name>/search/<key_name>/"
-Write : "http://<Domain Name>:<port_number>/<bucket_name>/write/<key_name>/"
-Delete : "http://<Domain Name>:<port_number>/<bucket_name>/delete/<key_name>/"
-Update : "http://<Domain Name>:<port_number>/<bucket_name>/update/<key_name>/"
+Search : "http://<Domain Name>:<port_number>/search/<key_name>/"
```

Comment:
with this name the endpoint seems to be so generics ,
try to make it specific as search_trust or search_org , 

do the same for all